index.html

```diff
@@ -8,6 +8,22 @@
 
 	<title>Rambox</title>
 
+	<script type='text/javascript'>
```

Comment:
This logic appears to have moved here: https://github.com/ramboxapp/community-edition/blob/18e1ed0809bfaea0a750219e814c1d0c3b1ce0ee/app/Application.js#L33